pkg/datapath/loader/base.go

```diff
@@ -145,11 +145,26 @@ func addENIRules(sysSettings []sysctl.Setting, nodeAddressing types.NodeAddressi
 		Val:       "2",
 		IgnoreErr: false,
 	})
-	if err := route.ReplaceRule(route.Rule{
+	// Delete old nodeport rule if it exists
+	oldRule := route.Rule{
 		Priority: linux_defaults.RulePriorityNodeport,
 		Mark:     linux_defaults.MarkMultinodeNodeport,
 		Mask:     linux_defaults.MaskMultinodeNodeport,
 		Table:    route.MainTable,
+	}
+	if err := route.DeleteRule(oldRule); err != nil {
+		if !os.IsNotExist(err) {
```

Comment:
`foo.IsBar()`-style functions are all deprecated in favor of `error.Is()`.
```suggestion
		if !errors.Is(err, fs.ErrNotExist) {
```